packages/vsphere/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.6.2"
+  changes:
+    - description: Added dimensions field mapping for virtualmachine datastream to support TSDB.
```

```suggestion
    - description: Add dimensions field mapping for virtualmachine datastream to support TSDB.
```

Reason: https://git.kernel.org/pub/scm/git/git.git/tree/Documentation/SubmittingPatches?h=v2.36.1#n181

> Describe your changes in imperative mood, e.g. "make xyzzy do frotz" instead of "[This patch] makes xyzzy do frotz" or "[I] changed xyzzy to do frotz", as if you are giving orders to the codebase to change its behavior.

Commit messages, changelog, PR titles, etc. ideally should follow this. This is the most recommended practice.

Also, it will match the tone of most changelog entries (if not all) we have made in the past i.e., `Migrate ...`, `Update ...`, etc.

---

Also, PR title needs a little tweaking i.e., s/Added/Add

These are nit change requests but having a consistent way of writing and doing is definitely recommended.